includes/ajax-handlers/class-wc-stripe-checkout-sessions-ajax-handler.php

```
@@ -126,4 +127,58 @@ public function create_checkout_session(): void {
+	 * Update a Stripe Checkout Session. Currently only used to update the line items.
+	 *
+	 * @return void
+	 */
+	public function update_checkout_session(): void {
+		try {
+			$is_nonce_valid = check_ajax_referer( 'wc_stripe_update_checkout_session_nonce', 'security', false );
+			if ( ! $is_nonce_valid ) {
+				throw new Exception( __( "We're not able to process this request. Please refresh the page and try again.", 'woocommerce-gateway-stripe' ) );
+			}
+
+			$session_id = isset( $_POST['checkout_session_id'] )
```

PHPMD (2.15.0)

[error] The variable $session_id is not named in camelCase. (undefined)

(CamelCaseVariableName)